fulp_modules/features/antagonists/bloodsuckers/code/powers/targeted/brawn.dm

```diff
@@ -17,11 +17,7 @@
 	power_activates_immediately = TRUE
 	prefire_message = "Select a target."
 
-/datum/action/cooldown/bloodsucker/targeted/brawn/can_use(mob/living/carbon/user, trigger_flags)
-	. = ..()
-	if(!.) // Default checks
-		return FALSE
-
+/datum/action/cooldown/bloodsucker/targeted/brawn/ActivatePower(trigger_flags)
```

Comment:
What's the difference between can_use and ActivatePower? When are they both called?

Reply:
this a leftover when i was trying to integrate can_use into IsAvailable but it didn't work out, it's better this way as it's more straight forward

Reply:
since both ifs put the ability on cooldown, otherwise we don't do checks beforehand that's it, looks cleaner